supertokens_python/recipe/passwordless/utils.py

```diff
@@ -126,15 +113,18 @@ def __init__(self,
 
 class ContactEmailOnlyConfig(ContactConfig):
     def __init__(self,
-                 create_and_send_custom_email: Callable[
-                     [CreateAndSendCustomEmailParameters, Dict[str, Any]], Awaitable[None]],
+                 create_and_send_custom_email: Union[Callable[
+                     [CreateAndSendCustomEmailParameters, Dict[str, Any]], Awaitable[None]], None] = None,
                  validate_email_address: Union[Callable[[
-                     str], Awaitable[Union[str, None]]], None] = None
+                     str], Awaitable[Union[str, None]]], None] = None,
+                 email_delivery: Union[EmailDeliveryConfig[TypePasswordlessEmailDeliveryInput], None] = None
                  ):
         super().__init__('EMAIL')
+        self.email_delivery = email_delivery
         if create_and_send_custom_email is None:
             self.create_and_send_custom_email = default_create_and_send_custom_email
         else:
+            warn("create_and_send_custom_email is depricated. Please use email delivery config instead")
```

Comment:
This is not the correct way to deprecate

Reply:
Using our util function `deprecated_warn` now. The IDEs don't highlight deprecated stuff for python.